web/www/missa/Polski/Tempora/Quad6-0.txt

```diff
@@ -9,6 +9,176 @@ Prelude
 Passio
 omit commemoratio
 
+[Prelude]
+#Poświęcenie Palm
+!Antyfona
+!Mt 21:9
+Hosanna Synowi Dawidowemu. Błogosławiony, który idzie w Imię Pańskie. 0 Królu Izraelski! Hosanna na wysokości.
+! Kapłan stojąc, po stronie Lekcji, nie odwracając się do ludu, mówi:
+$Dominus vobiscum
+v. Boże, którego czcić i kochać powinniśmy, rozmnóż w nas niewypowiedzianej łaski Twojej dary, a który przez śmierć Syna Swego dałeś nam nadzieję osiągnięcia tego, w co wierzymy, spraw, abyśmy przez Jego zmartwychwstanie mogli rzeczywiście dojść tam, dokąd dążymy.
+$Qui tecum
+_
+_
+! Następnie Subdiakon ze zwykłego miejsca śpiewa lekcję, skończywszy całuje rękę Kapłana.
+!!Lekcja
+Czytanie z Księgi Wyjścia
+!Wj 15:27; 16:1-7
+Onych dni: Przyszli do Elim synowie Izraelowi, gdzie było dwanaście źródeł wód i siedmdziesiąt palm, i obozem stanęli przy wodzie. I ruszyli się z Elim, i przyszło wszystko mnóstwo synów Izraelowych na puszczą Syn, która jest między Elim i Synai, piętnastego dnia miesiąca wtórego potem, jako wyszli z ziemie Egipskiéj. I szemrało wszystko zgromadzenie synów Izraelowych przeciw Mojżeszowi i Aaronowi na puszczy. I mówili synowie Izraelscy do nich: O byśmy byli pomarli od ręki Pańskiéj w ziemi Egipskiéj, gdyśmy siedzieli nad garncy mięsa, i jedliśmy chleb w sytości: czemuście nas wywiedli na tę puszczą, abyście wszystko mnóstwo głodem pomorzyli? I rzekł Pan do Mojżesza: oto ja spuszczę wam chleb z nieba: niechaj wychodzi lud, a zbiera, coby dosyć było, na każdy dzień: abym go doświadczył, jeźliż chodzi w zakonie moim, albo nie? A dnia szóstego niechaj nagotują, coby wnieśli: a niech będzie tyle dwoje, niż co zbierać zwykli na każdy dzień. I rzekli Mojżesz i Aaron do wszystkich synów Izraelowych: W wieczór poznacie, że was Pan wyprowadził z ziemie Egipskiéj: A rano ujrzycie chwałę Pańską.
+$Deo gratias
+_
+_
+! Następnie śpiewa się Graduał
+!!Graduał
+!J 11:47-49, 50, 53
+R. Zebrali tedy najwyżsi kapłani i Faryzeuszowie radę i mówili: Cóż uczynimy, albowiem Ten człowiek wiele cudów czyni? Jeśli Go tak zaniechamy, wszyscy weń uwierzą:
+* I przyjdą Rzymianie i wezmą nasze miejsce i naród.
+V. A jeden z nich, Kaifasz imieniem, będąc najwyższym kapłanem onego roku prorokował im mówiąc: Pożyteczno wam jest, żeby jeden człowiek umarł za lud, a nie wszystek naród zginął. Od onego tedy dnia umyślili, aby Go zabili, mówiąc:
+* I przyjdą Rzymianie i wezmą nasze miejsce i naród.
+
+!Vel aliud
+!Mt 28:39, 41
+R. Na górze Oliwnej modlił się do Ojca: Ojcze, jeśli można, niechaj odejdzie odemnie ten kielich.
+* Duch wprawdzie jest ochotny, ale ciało mdłe, bądź wola Twoja.
+V. Czuwajcie i módlcie się, abyście nie weszli w pokusę.
+* Duch wprawdzie jest ochotny, ale ciało mdłe, bądź wola Twoja.
+_
+_
+! Interim dum cantatur Responsorium, Diaconus ponit librum Evangeliorum super Altare: et Sacerdos, ministrante Diacono naviculam, ponit incensum in thuribulo. Deinde Diaconus dicit: Munda cor meum, accipit librum de Altari, petit benedictionem a Sacerdote:
```

Comment:
There is some Latin here. I think that these texts are translated in Ordinarium.